webui/common/src/store/config/types.tsx

```diff
@@ -11,7 +11,7 @@
 
 import { AxiosResponse } from 'axios';
 
-import { IConfigTriple } from '../../constants';
+import { IConfigTriple } from '../../../../master/src/constants';
```

Comment:
`IConfigTriple` was moved to `common` right? in the `common` project try not to import anything from `master` or `worker`